### PR TITLE
[feat] add karma grafana dashboard

### PR DESCRIPTION
**Pull request by dongjiang1989**
### What type of PR is this?
/kind feature

- [x] Unittest
- [x] Coverage 80%+
- [x] Build Success

### CODE REVIEW

**Comment by dongjiang1989** [comment]
support karma dashboard for alertmanager.
WeChatbdc25e9562a3cc3a8a6a70eff2a19a75

File: metrics/charts/grafana/values.yaml [hunk 1]
```diff
@@ -686,6 +686,11 @@ grafana.ini:
     url: https://grafana.net
   server:
     domain: "{{ if (and .Values.ingress.enabled .Values.ingress.hosts) }}{{ .Values.ingress.hosts | first }}{{ else }}''{{ end }}"
+  enable_alpha: true
```
Comment (imlense) [inline metrics/charts/grafana/values.yaml]: miss `ini` config  items key

**Review by imlense — COMMENTED** [review]
LGTM

File: metrics/charts/grafana/values.yaml (hunk 1 above)
Comment (dongjiang1989) [reply metrics/charts/grafana/values.yaml]: fixed! Please recheck! @imlense

**Pull request merged**